perception/lidar_transfusion/include/lidar_transfusion/visibility_control.hpp

```diff
@@ -0,0 +1,37 @@
+// Copyright 2024 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LIDAR_TRANSFUSION__VISIBILITY_CONTROL_HPP_
```

Comment:
Is there any reason for this file?
I don't think we support anything other than linux

Reply:
My motivation was fact that some of Autoware packages already consider `visibility_control`. The Coding guidelines in Autoware documentation section do not cover it yet, so I assume the other way: packages may missing this header. If this header is somehow a problem for us, let me know

Reply:
Wow, I  was not aware of the details
https://gcc.gnu.org/wiki/Visibility

Let's definitively keep this :+1: